spec/models/captioner_spec.rb

```diff
@@ -0,0 +1,8 @@
+require_relative '../../lib/captioner'
+
+describe Captioner do
+  it 'produces WebVTT' do
+    tei = File.read(__dir__ + '/../fixtures/transcript/mock-transcript.xml')
+    expect(Captioner.from_tei(tei)).to eq(File.read(__dir__ + '/../fixtures/transcript/mock-transcript.vtt'))
+  end
+end
```

Comment:
Can you wrap unit tests in `describe method_name`? It makes it more explicit about exactly what is being tested. E.g...

```ruby
describe Captioner do
  describe '.from_tei' do
    it 'produces WebVTT' do
```